components/Conversations/Detail/ConversationDetail/NoAudioDetected.tsx

```diff
@@ -1,54 +1,50 @@
-import React from 'react';
-import GreyShieldTick from '../Icon/GreyShieldTick';
-import BigSoundIcon from '../Icon/BigSoundIcon';
-import TrashIcon from '../Icon/TrashIcon';
+import React from 'react'
+import GreyShieldTick from '../Icon/GreyShieldTick'
+import BigSoundIcon from '../Icon/BigSoundIcon'
+import { Trash } from 'iconsax-react'
+// import TrashIcon from '../Icon/TrashIcon';
 
 const NoAudioDetected: React.FC = () => (
-  <div className="w-[840px] h-[1829px] relative font-inter">
+  <div className="font-inter relative h-[1829px] w-[840px]">
```

Comment:
if we want inter font should't we just pass that to main `layout.tsx` ?

Reply:
Yeah this is a more general cleanup task im going to hand off